src/js/control-bar/control-bar.js

```diff
@@ -68,6 +69,7 @@ ControlBar.prototype.options_ = {
     'descriptionsButton',
     'subtitlesButton',
     'captionsButton',
+    'subCapsButton',
```

Comment:
this should be `subsCapsButton` took me forever to figure out why it wasn't working for me, haha